packages/xml-views-validation/api.d.ts

```diff
@@ -0,0 +1,26 @@
+import { XMLDocument } from "@xml-tools/ast";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+
+export function validateXMLView(opts: {
+  model: UI5SemanticModel;
+  xmlView: XMLDocument;
+}): UI5XMLViewIssue[];
+
+export interface UI5XMLViewIssueStructure {
+  kind: string;
+  message: string;
+  severity: "hint" | "info" | "warn" | "error";
+  range: { start: number; end: number };
```

Comment:
Are `start` and `end` offsets? Could you name them as such (`startOffset` and `endOffset`)?

Reply:
I thought it may be redundant but now I see in LSP it is a "Position" for start/end.
Will Change

Reply:
I will change the property name to `offsetRange` to avoid confusion with LSP's range
and to avoid duplication.